docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -108,7 +111,7 @@ Checklist:
       - [ ] IPFS Discord #ipfs-chatter
       - [ ] FIL Slack #ipfs-chatter
       - [ ] Matrix https://matrix.to/#/#ipfs-chatter:ipfs.io
-    - [ ] Mention [early testers](https://github.com/ipfs/go-ipfs/tree/master/docs/EARLY_TESTERS.md) in the comment under the release issue ([example](https://github.com/ipfs/kubo/issues/9237#issuecomment-1258072509)).
+    - [ ] Mention [early testers](https://github.com/ipfs/go-ipfs/tree/master/docs/EARLY_TESTERS.md) in the comment under the release issue ([example](https://github.com/ipfs/kubo/issues/9319#issuecomment-1311002478)).
```

Comment:
I updated the link because in the latest comment I added context for why folks were being pinged and how they get removed.

For example, while @b5 is still absolutely in the community, QRI is no-more and I don't think need to be pinged about Kubo releases.